src/OpenLoco/OpenLoco.cpp

```diff
@@ -1158,6 +1159,14 @@ namespace OpenLoco
     // 0x00406D13
     void main()
     {
+        if (!OpenLoco::platform::isRunningInWine())
+        {
+            crash_init();
+        }
+        else
+        {
+            std::printf("Detected wine, not installing crash handler as it doesn't provide useful data. Consider using native builds of OpenLoco instead.\n");
```

Comment:
the rest of this function uses std::cout. Stick to that.

Reply:
Alternatively, `Console::log`

Reply:
I was actually basing my decision on other log calls in this file:
https://github.com/OpenLoco/OpenLoco/blob/bfd99e849d231f25cb3a93992ac34f4f070efa8f/src/OpenLoco/OpenLoco.cpp#L948-L953
https://github.com/OpenLoco/OpenLoco/blob/bfd99e849d231f25cb3a93992ac34f4f070efa8f/src/OpenLoco/OpenLoco.cpp#L910
https://github.com/OpenLoco/OpenLoco/blob/bfd99e849d231f25cb3a93992ac34f4f070efa8f/src/OpenLoco/OpenLoco.cpp#L357
https://github.com/OpenLoco/OpenLoco/blob/bfd99e849d231f25cb3a93992ac34f4f070efa8f/src/OpenLoco/OpenLoco.cpp#L918